pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -1504,14 +1545,58 @@ func isDeviceOpened(deviceToDetach AttachedVolume, mounter mount.Interface) (boo
 	return deviceOpened, nil
 }
 
-func isMigrated(vpm *volume.VolumePluginMgr, spec *volume.Spec) bool {
+func isNodeMigrated(og *operationGenerator, pluginName string, nodeName types.NodeName) (bool, error) {
+	var err error
+	var nodeInfo *csiapi.CSINodeInfo
+
+	if og.csiNodeInfoHasSynced() && utilfeature.DefaultFeatureGate.Enabled(features.CSINodeInfo) {
+		nodeInfo, err = og.csiNodeInfoLister.Get(string(nodeName))
+		if err != nil {
+			return false, err
+		}
+
+	} else if utilfeature.DefaultFeatureGate.Enabled(features.CSINodeInfo) {
+		glog.Warningf("CSINodeInfo informer not synced, have you made sure the CSINodeInfo CRD is installed. If so, this error should not appear again after ~1 minute")
```

Comment:
The warning message can be made a little better: `have you made sure the CSINodeInfo CRD is installed.` => `please check CSINodeInfo CRD is installed.`

and `this error should not appear again` => `this warning should not appear again`